libs/domain/search/src/renderers/facet.providers.ts

```diff
@@ -24,22 +25,31 @@ export const facetProviders = [
           options: FacetMappingOptions,
           selectListener: (e: CustomEvent<SelectFacetEventDetail>) => void
         ): TemplateResult => {
+          if (featureVersion >= '1.2' && facet.type === FacetType.Range) {
+            return html` <oryx-search-range-facet
+              @oryx.select=${selectListener}
+              .name=${facet.name}
+              ?open=${options.open}
+              ?disableClear="${!options.enableClear}"
+            ></oryx-search-range-facet>`;
+          }
+
           return html`
             <oryx-search-facet
               @oryx.select=${selectListener}
               .name=${facet.name}
               .renderLimit=${options.renderLimit}
               .open=${options.open}
-              .enableClear="${options.enableClear}"
-              .multi=${facet.multiValued}
+              ?disableClear="${!options.enableClear}"
```

Comment:
BC; we should also add the former field for as long as we're < 1.2.

`.enableClear=${ifDefined(featureVersion >= '1.2' ? undefined : options.enableClear}}`